### PR TITLE
💄Added Delete Functionality to the End Block #CLM-325

### PR DESCRIPTION
**Pull request by Beryl-ajuoga**
# Description
Added the delete functionality to the end block on user story

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] My changes generate no new warnings
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by LemmyMwaura — CHANGES_REQUESTED** [review]
The functionality has not yet been added. This needs to have some logic to delete the block. But also attach a screenshot in your readme.

**Review by ReaganCn — CHANGES_REQUESTED** [review]
Good implementation but you can update the styling of the start and endblock to match the design. Currently with the old design it doesnt look great.

**Comment by Beryl-ajuoga** [comment]
> The functionality has not yet been added. This needs to have some logic to delete the block. But also attach a screenshot in your PR description.

okay

**Comment by Beryl-ajuoga** [comment]
> The functionality has not yet been added. This needs to have some logic to delete the block. But also attach a screenshot in your PR description.

okay

**Comment by Beryl-ajuoga** [comment]
> Good implementation but you can update the styling of the start and endblock to match the design. Currently with the old design it doesnt look great.

okay

File: libs/features/convs-mgr/stories/blocks/library/main/src/lib/components/block/block.component.html [hunk 1]
```diff
@@ -113,10 +113,13 @@ <h4 class="block-title"> {{ blockTitle | transloco }}</h4>
...
             </app-open-ended-question-block>
         </div>
-        <div *ngSwitchCase="endStoryAnchor">
-            <app-end-anchor [id]="id" [block]="block" [endStoryAnchorForm]="blockFormGroup" [jsPlumb]="jsPlumb" class="block">
-            </app-end-anchor>
-        </div>
+        <div *ngSwitchCase="endStoryAnchor" class="outer">
+                <div class="block-title-container2"> 
+                    <app-end-anchor [id]="id" [block]="block" [endStoryAnchorForm]="blockFormGroup" [jsPlumb]="jsPlumb" class="block">
+                    </app-end-anchor>
```
Comment (ReaganCn) [inline libs/features/convs-mgr/stories/blocks/library/main/src/lib/components/block/block.component.html:119]: Receive the delete event over here and then call the delete method in this component with the necessary parameters. [https://ultimatecourses.com/blog/component-events-event-emitter-output-angular-2](link)

**Review by LemmyMwaura — APPROVED** [review]

**Pull request merged**